build.sh

```diff
@@ -862,7 +906,7 @@ function build_micro_cpp()
     echo -e "${INFO_TAG} Compiling perftest_cpp"
     cd "${classic_cpp_folder}"
 
-    cmake_generate_command="${CMAKE_EXE} -DCMAKE_BUILD_TYPE=${RELEASE_DEBUG} -G \"Unix Makefiles\" -B./perftest_build -H. -DRTIME_TARGET_NAME=${platform} -DPLATFORM_LIBS=\"dl;m;pthread;${additional_included_libraries}\""
+    cmake_generate_command="${CMAKE_EXE} ${additional_cmake_options} -D CMAKE_BUILD_TYPE=${RELEASE_DEBUG} -G \"Unix Makefiles\" -B./perftest_build -H. -DRTIME_TARGET_NAME=${platform} -DPLATFORM_LIBS=\"dl;m;pthread;${additional_included_libraries}\""
```

Comment:
@salvacorts, I am wondering if we should expose or not the custom CRC option. I mention this because it would involve also allowing the user to add this additiona_cmake_options, but also the need of modifying the additional_defines. What is your take on this?